source-repo-scripts/README.md

```diff
@@ -39,6 +39,35 @@ its dependencies on fortress using the `chapulina/fortress` branch for `docs`:
 "chapulina/fortress"
 ```
 
+### merge_forward_pull_request.bash
+
+The script will open a pull request for a forward port after the user has
+already created the branch, run `git merge`, resolved conflicts, and committed
+the result.
+
+#### Usage
+
+Requires the 'gh' CLI to be installed.
+```bash
+./merge_forward_pull_request.bash <from_branch> <to_branch>
+```
+
+#### Example
+
+To merge `ign-rendering6` to `main`:
+
+```bash
+cd ign-rendering
```

Comment:
Maybe for future work but these steps could probably be part of a script too until the point of the manual intervention for the conflict resolution.

Reply:
@chapulina suggested the same https://github.com/gazebo-tooling/release-tools/pull/720#discussion_r868323853, and we decided to address in a follow-up pull request